.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Thank you for taking the time to contribute to Odin-pantry. Please complete this entire template. -->
+
+## Because
```

Comment:
Hmmm, this is perhaps confusing to someone not used to PRs and using a PR template. 

What do you think about giving this section a title such as "PR reason", or "What this PR introduces" or some other title...

Reply:
Sounds reasonable. I'll update that.
This is an adaptation of TOP's PR template, so I didn't change a lot of the "legalistic" wording. I'll do my best to make it more beginner friendly given the purpose of this project.

Reply:
Updated a major chunk of the wording and added examples in every section to help explain things. Let me know what you think.